src/ReactiveDomain.Users/Sevices/UserSvc.cs

```diff
@@ -0,0 +1,109 @@
+﻿using ReactiveDomain.Foundation;
+using ReactiveDomain.Messaging;
+using ReactiveDomain.Messaging.Bus;
+using ReactiveDomain.Users.Domain;
+using ReactiveDomain.Users.Messages;
+
+namespace ReactiveDomain.Users.Sevices
+{
+    /// <summary>
+    /// The service that fronts the User aggregate.
+    /// </summary>
+    public class UserSvc :
+        IHandleCommand<UserMsgs.CreateUser>,    
+        IHandleCommand<UserMsgs.Deactivate>,
+        IHandleCommand<UserMsgs.Activate>,
+        IHandleCommand<UserMsgs.UpdateUserDetails>,
+        IHandleCommand<UserMsgs.MapToAuthDomain>,
+        IHandleCommand<UserMsgs.AddClientScope>,
+        IHandleCommand<UserMsgs.RemoveClientScope>
+    {
+        private readonly CorrelatedStreamStoreRepository _repo;
+
+        /// <summary>
+        /// Create a service to act on User aggregates.
+        /// </summary>
+        /// <param name="repo">The repository for interacting with the EventStore.</param>
+        /// <param name="bus">The dispatcher.</param>
+        public UserSvc(IRepository repo)
```

Comment:
Should we allow injection of a cache?

Reply:
I might be nice, but that's a bit more complicated than it first appears.